RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -664,6 +664,7 @@ StatusCode AugmentClustersFCCee::execute([[maybe_unused]] const EventContext &ev
             double theta2_E_3Bin = 0.;
             double theta_E_3Bin = 0.;
             double sum_E_3Bin = 0.;
+            double sum_weightLog_3Bin = 0.;
```

Comment:
why declaring/calculating the wightlog variables here and not later inside the if (m_do_widthTheta_logE_weights) {..} block? Maybe the compiler is smart enough to optimise this code, but it makes anyway the code logic a bit harder to follow.